backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap_classic_128.cuh

```diff
@@ -93,8 +97,7 @@ __global__ void __launch_bounds__(params::degree / params::opt)
   // Put "a" in [0, 2N[
   InputTorus a_hat = 0;
   modulus_switch<InputTorus>(block_lwe_array_in[lwe_iteration], a_hat,
-                             params::log2_degree +
-                                 1); // 2 * params::log2_degree + 1);
+                             log_modulus); // 2 * log_modulus);
```

Comment:
I don't understand the use of the comment `2 * log_modulus`

Also it used to be `2 * params::log2_degree + 1` which is not equal to `2 * log_modulus`